src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const copyRobot = {};
+
+  for (const key in robot) {
+    if (copyRobot.hasOwnProperty(robot[key])) {
+      return null;
+    } else {
+      copyRobot[robot[key]] = key;
+    }
```

Comment:
Якщо в попередньому if є return тоді else вже не потрібен
```suggestion
    if (copyRobot.hasOwnProperty(robot[key])) {
      return null;
    }
    
    copyRobot[robot[key]] = key;
```